packages/ui-keyring/src/options/index.ts

```diff
@@ -129,10 +133,25 @@ class KeyringOption implements KeyringOptionInstance {
       });
   }
 
+  private addContracts (keyring: KeyringStruct, options: KeyringOptions): void {
+    const available = keyring.contracts.subject.getValue();
+
+    Object
+      .values(available)
+      .filter(({ json }: SingleAddress) => {
+        return !json.meta.isRecent;
```

Comment:
We don't need the isRecent check here. (Only for addresses where we "magically" add them)